pkgs/by-name/ac/aces_container/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aces-container";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "ampas";
+    repo = "aces_container";
+    rev = "v${version}";
+    hash = "sha256-luMqXqlJ6UzoawEDmbK38lm3GHosaZm/mFJntBF54Y4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = {
+    description = "Reference Implementation of SMPTE ST2065-4";
+    homepage = "https://github.com/ampas/aces_container";
+    license = lib.licenses.unfree; # FIXME: nix-init did not find a license
```

Comment:
This needs a new entry in `licenses.nix`, then refer to it here.
SPDX info is available at https://spdx.org/licenses/AMPAS.html